projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -121,5 +122,25 @@ export class GoCalendarComponent implements OnDestroy, OnInit {
     }
     this.currentMonth = this.selectedDate.getMonth();
     this.updateYear(this.selectedDate.getFullYear());
+    if (this.appendTo === 'body') {
+      setTimeout(() => {
+        const calenderBody: any = document.getElementsByClassName(
+          'append-body'
+        )[0];
+        const calenderBodyPosition: any = document
+          .querySelector('.append-body')
+          .getBoundingClientRect();
+        Object.assign(calenderBody.style, {
+          height: `${calenderBodyPosition.height}px`,
+          bottom: `${calenderBodyPosition.bottom}px`,
+          top: `${calenderBodyPosition.top}px`,
+          right: `${calenderBodyPosition.right}px`,
+          width: `${calenderBodyPosition.width}px`,
+          left: `${calenderBodyPosition.left}px`,
+          visibility: 'visible',
+        });
```

Comment:
Do we have to assign all of these properties when doing this? Specifically `height`/`width`?

Reply:
If I have removed few properties so some alignment issue is coming. For that reason, I have added those properties. Can you please suggest to me if anything is not required. So I will remove those.